src/Query/ResultPrinters/CategoryResultPrinter.php

```diff
@@ -229,7 +229,8 @@ private function getContents( QueryResult $res, $outputMode ) {
 
 		// Make label for finding further results
 		if ( $this->linkFurtherResults( $res ) ) {
-			$contents[$last_letter][] = $this->getFurtherResultsLink( $res, $outputMode )->getText( SMW_OUTPUT_WIKI, $this->mLinker );
+			$index = $last_letter ?? : $first_letter;
```

Comment:
As far as I can tell this is invalid PHP syntax

![image](https://user-images.githubusercontent.com/146040/184383854-a0b127ab-51cc-42bf-9b4a-4fa1cea59ee2.png)

Reply:
sorry, fixed!